ProjectGroupA/src/main/java/com/fundation/search/model/Search.java

```diff
@@ -90,4 +109,37 @@ private boolean filterResults(String inputline, SearcherCriteria file) {
 			return true;
 		} else return false;
 	}
+	
+	/**
+	 * Method which compares the size specified in the criteria and the files found.
+	 */
+	private boolean matchSizeCriteria (String inputline, SearcherCriteria criteria) {
+		boolean res = false;
+		if (! criteria.getSize().equals("0")) {
+			long cSize = Long.parseLong(criteria.getSize());
+			
+			File tFile = new File (inputline);
+			long tFileSize = tFile.length();
+			
+			switch (criteria.getOperator()) {
```

Comment:
use the following format to switch statement: 
switch (condition) {
case ABC:
    statements;
    /* falls through */
case DEF:
    statements;
    break;
case XYZ:
    statements;
    break;
default:
    statements;
    break;
}

each word should be aligned with switch word

Reply:
done, reviewed the link that Paolo shared with us.